ArchiSteamFarm/Bot.cs

```diff
@@ -134,6 +134,7 @@ public sealed class Bot : IDisposable {
 		[JsonProperty]
 		private string AvatarHash;
 
+		private string CachedTradeToken;
```

Comment:
With new AH way of getting token, you should not cache it as it may still change during runtime and there is no longer a reason to do so, as trade token is not used often enough and AH responses are instant.